src/Summoner/Project.hs

```diff
@@ -153,10 +154,11 @@ generateProject projectName Targets{..} = do
     test   <- decisionToBool isTest "tests"
     bench  <- decisionToBool isBenchmark "benchmarks"
 
-    putStrLn "Latest GHCs: 7.10.3 8.0.2 8.2.2"
-    putStrLn $ "The project will be created with the latest resolver for GHC-" ++ T.unpack defaultGHC
-    testedVersions <- T.words <$>
+    T.putStrLn "Latest GHCs: 7.10.3 8.0.1 8.0.2 8.2.2"
```

Comment:
Maybe here it's better to tell something like `Latest supported by summoner GHCs`? 🤔

Reply:
I don't really like this plain text, as we should be keeping in mind changing it when we change the list of supported versions..

Reply:
@vrom911 Maybe we could have something like that `allGhcVersions = [minBound .. maxBound]` to not forget to update this list.